databases/relational-databases.md

```diff
@@ -37,35 +34,27 @@ To understand, query, and insert information into a relational database, a techn
 - Explain the concept of a relational database and the use of schemas to enforce structure
 - Execute basic SQL commands for CRUD with a database
 - Perform basic selects and joins to find the requested information
```

Comment:
Can you remove all references to "join"? They haven't learned it yet in the curriculum.

Reply:
Sure! on it.

Reply:
Done!